pkg/store/bucket_test.go

```diff
@@ -1259,11 +1280,17 @@ func benchBucketSeries(t testutil.TB, samplesPerSeries, totalSeries int, request
 
 	var bCases []*storetestutil.SeriesCase
 	for _, p := range requestedRatios {
+		allCut := int(p * float64(totalSeries*samplesPerSeries))
```

Comment:
Let's fix naming I agree it's confusing. This is the number of series we expect to be returned for our request. Feel free to propose rename :hugs:

Reply:
Why this is the number of expected series, not expected total samples?